contracts/solidity/mocks/MockAggregator.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract MockAggregator {
+  int256 public s_answer;
+
+  function setLatestAnswer(int256 answer) public {
+    s_answer = answer;
+  }
+
+  function latestAnswer() public view returns (int256) {
```

Comment:
This should be latestRoundData I think

https://github.com/smartcontractkit/chainlink-starknet/blob/develop/contracts/solidity/emergency/StarknetValidator.sol#L184

Reply:
@sdrug are you waiting on review? I think this still needs to be changed. The line above has been changed to https://github.com/smartcontractkit/chainlink-starknet/blob/develop/contracts/solidity/emergency/StarknetValidator.sol#L145 now